logging/logger_manager.go

```diff
@@ -0,0 +1,53 @@
+package logging
+
+import (
+	"fmt"
+	"sync"
+)
+
+type loggerRegistry struct {
+	mu      sync.Mutex
```

Comment:
Can we use a RWMutex so that reads don't block each other?

Reply:
yes good suggestion